portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,24 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
```

Comment:
The walkthroughs stick with just using one webpage, so they don't mention this, but this is the magic line that lets you call JS functions from script.js here in index.html. 

You went above and beyond with extra webpages, so you just need to add this in the <head> section to any html files where you want to call functions from script.js